app/Bundle.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Bundle extends Model
+{
+    /**
+     * The attributes that are mass assignable.
+     *
+     * @var array
+     */
+    protected $fillable = [
+        'entitlement',
+        'allocated_at',
+        'centre_id',
+        'family_id'
+    ];
+
+    protected $rules = [
+    ];
+
+    /**
+     * The attributes that should be cast to native types.
+     *
+     * @var array
+     */
+    protected $casts = [
+    ];
+
+    /**
+     * The attributes that should be hidden for arrays.
+     *
+     * @var array
+     */
+    protected $hidden = [
+    ];
+
+    /**
+     * The attributes to append to the model's array form.
+     *
+     * @var array
+     */
+    protected $appends = [
+    ];
+
+    /**
+     * These are turned into Date objects on get
+     *
+     * @var array
+     */
+    protected $dates = [
+        'created_at',
+        'updated_at',
+        'allocated_at'  // When it was handed out.
+    ];
+
+    /**
+     * Return the Centre it was allocated to
+     *
+     * @return \Illuminate\Database\Eloquent\Relations\BelongsTo
+     */
+    public function centre()
```

Comment:
If we don't use these relationships - do we want to define them? In this PR - they seem only to be used in tests - but I might be missing something.